source-similarity-matrix.py

```diff
@@ -15,6 +16,10 @@
 logger = get_logger()
 
 
+def embed(input):
+    return model(input)
```

Comment:
is this going to pick up the latest value assigned to model in the driver code? We could alternatively pass the model to the embed function, just to make it clearer what the model variable is pointing at (since we have different models now)

Reply:
If you see change in line 78 in the source-similarity-matrix.py we are passing model so it will get the concerned model for each region.

Reply:
I know this works, my concern was more stylistic. We have a function that uses a global variable that is only defined later in the code. I think I added it like that in the beginning, but kept thinking we should change it at some point. But if it doesn't bother you much we can leave as is 👍